lib/util/include/hse_util/data_tree.h

```diff
@@ -338,7 +353,6 @@ dt_tree_emit_path(char *path);
 void
 dt_tree_emit_pathbuf(char *path, char *buf, u32 buflen);
 
-dt_field_t
-dt_get_field(const char *field);
+#endif
```

Comment:
Are those functions only used by unit tests?  In which case #if MTF-whatever... ???

Reply:
Yeah, that might be useful, at least they're now out of the release build :-)

Reply:
If they're not used anywhere, I'd just get rid of them.